.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  pull_request:
+    branches:
+      - master
+
+name: Test UI
+
+jobs:
+  release_assets:
+    name: Test UI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.5'
+      - name: Lookup yarn cache
+        id: yarn_cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn_cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - uses: actions/cache@v1
+        with:
+          path: ~/.tiup/components
+          key: ${{ runner.os }}-tiup
+          restore-keys: |
+            ${{ runner.os }}-tiup
+      - name: Install and run TiUP
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+          source /home/runner/.profile
+          tiup update --nightly --all
+          tiup playground nightly &
```

Comment:
I tried:

```
~/.tiup/bin/tiup update --nightly --all
~/.tiup/bin/tiup playground nightly &
```

to avoid `source /home/runner/.profile` hard-code, but it doesn't work. I can try it again later.